src/Mealz/MealBundle/Resources/views/Meal/partials/meal.html.twig

```diff
@@ -13,8 +14,9 @@
     {% endif %}
 {% endif %}
 
-<div class="meal-row combined-meal"
+<div class="meal-row{{ combinedDish is same as('1') ? 'combined-meal' }} "
     {%- if available -%}
```

Comment:
We already talked about it. :-) I used data-fields too for meals that are not available. Please use isCombinedMeal for the combined meal related stuff.